webapp/src/cljs/lipas/ui/sports_sites/views.cljs

```diff
@@ -243,13 +245,42 @@
            read-only? sub-headings? address-required?]
     :or   {address-required? true}}]
   (r/with-let [no-address? (r/atom (= "-" (:address display-data)))]
-    (let [locale (tr)]
+    (let [locale (tr)
+          cities (<== [:lipas.ui.sports-sites.subs/cities-list])
+          mode (-> (<== [:lipas.ui.map.subs/mode*]))
+          lipas-id (-> mode :lipas-id)
+          selected-sports-site-geoms (-> (<== [:lipas.ui.sports-sites.subs/sports-site lipas-id])
```

Comment:
Please move this logic from `view` to `subs`. Create a new sub for e.g. `selected-site-first-point`

Reply:
Fixed!